From 0475aa2bc952f0791f3b18248534df099ad4d1ae Mon Sep 17 00:00:00 2001
From: Danny Kukawka <danny.kukawka@web.de>
Date: Fri, 16 Apr 2010 12:12:08 +0200
Subject: [PATCH 31/33] add a Cowon iAudio S9 as USB music player

Added an adopted patch from James Ogley <riggwelter@opensuse.org> to
add a Cowon iAudio S9 as USB music player. (bnc#552488)
---
 .../10freedesktop/10-usb-music-players.fdi         |   14 ++++++++++++++
 1 files changed, 14 insertions(+), 0 deletions(-)

diff --git a/fdi/information/10freedesktop/10-usb-music-players.fdi b/fdi/information/10freedesktop/10-usb-music-players.fdi
index d2f0adf..9b908d5 100644
--- a/fdi/information/10freedesktop/10-usb-music-players.fdi
+++ b/fdi/information/10freedesktop/10-usb-music-players.fdi
@@ -737,6 +737,20 @@
 	    <append key="portable_audio_player.audio_folders" type="strlist">RECORDS/LINEIN/</append>
 	    <append key="portable_audio_player.audio_folders" type="strlist">RECORDS/VOICE/</append>
 	  </match>
+	  <!-- iAudio S9 -->
+          <match key="@storage.originating_device:usb.product_id" int="0x900">
+	   <addset key="portable_audio_player.access_method.protocols" type="strlist">storage</addset>
+	   <append key="portable_audio_player.output_formats" type="strlist">application/ogg</append>
+  	   <append key="portable_audio_player.output_formats" type="strlist">audio/x-ms-wma</append>
+	   <append key="portable_audio_player.output_formats" type="strlist">audio/flac</append>
+	   <append key="portable_audio_player.output_formats" type="strlist">audio/x-wav</append>
+	   <append key="portable_audio_player.input_formats" type="strlist">audio/mpeg</append>
+	   <append key="portable_audio_player.playlist_format" type="strlist">audio/x-mpegurl</append>
+	   <append key="portable_audio_player.audio_folders" type="strlist">MUSIC/</append>
+	   <append key="portable_audio_player.audio_folders" type="strlist">RECORDINGS/FM RADIO/</append>
+	   <append key="portable_audio_player.audio_folders" type="strlist">RECORDINGS/LINE-IN/</append>
+	   <append key="portable_audio_player.audio_folders" type="strlist">RECORDINGS/MIC/</append>
+	  </match>
 	</match>
 
 	<!-- Archos -->
-- 
1.7.1

